app/main.py

```diff
@@ -1,8 +1,32 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: dict) -> list:
+    person_list = [
+        Person(person_info["name"], person_info["age"])
+        for person_info in people
+    ]
+    Person.people = {person.name: person for person in person_list}
```

Comment:
There is no need in this line, this operation is performed in the `__init__` of the `Person` class when you initialize a new instance.